README.md

```
@@ -39,57 +39,37 @@ The main way to install EVOKS and its depending services is to build and run the
-2. (until further notice) checkout development branch
-    ```
-    git checkout development
-    ```
 2. Copy .example.env and rename to .env
 3. Open .env and change the variables if desired, see
 [Overview of variables](##Variables).
 
-    **Caution** Change of Fuseki and Postgres credentials is strongly recommended, especially if not installed and used only on a local computer without access from other people
+    **Caution** Change of Fuseki and Postgres credentials is strongly recommended, especially if it will be accessible from outside
 
+4. Open evoks/evoks/settings.py and change the SECRET_KEY. Use of a key generator like https://djecrety.ir/ is strongly recommended.
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)